.gitlab-ci.yml

```diff
@@ -56,7 +56,7 @@ build:
 build_artifacts:
     stage: build
     only:
-        - /^v[0-9]+\..*/
+        - /^(v\d+\.\d+\.\d+(-[a-zA-Z]+\.\d)?|main)$/
```

Comment:
Tags and commits from main branch are allowed.

v1.0.0
v.0.0.0-rc.1
v.0.0.0-beta.1
main